src/SprykerEco/Zed/Unzer/Business/ApiAdapter/Mapper/UnzerAuthorizePaymentMapper.php

```diff
@@ -109,4 +112,35 @@ public function mapUnzerApiAuthorizeResponseTransferToUnzerPaymentTransfer(
             ->setPaymentResource((new UnzerPaymentResourceTransfer())->setId($unzerApiAuthorizeResponseTransfer->getTypeId()))
             ->setRedirectUrl($unzerApiAuthorizeResponseTransfer->getRedirectUrl());
     }
+
+    /**
+     * @param \Generated\Shared\Transfer\UnzerApiErrorResponseTransfer $unzerApiErrorResponseTransfer
+     * @param \Generated\Shared\Transfer\UnzerPaymentTransfer $unzerPaymentTransfer
+     *
+     * @return \Generated\Shared\Transfer\UnzerPaymentTransfer
+     */
+    public function mapUnzerApiErrorResponseTransferToUnzerPaymentTransfer(
+        UnzerApiErrorResponseTransfer $unzerApiErrorResponseTransfer,
+        UnzerPaymentTransfer $unzerPaymentTransfer
+    ): UnzerPaymentTransfer {
+        foreach ($unzerApiErrorResponseTransfer->getErrors() as $unzerApiResponseErrorTransfer) {
+            $unzerPaymentErrorTransfer = $this->createUnzerPaymentErrorTransfer($unzerApiResponseErrorTransfer);
+
+            $unzerPaymentTransfer->addError($unzerPaymentErrorTransfer);
+        }
+
+        return $unzerPaymentTransfer;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\UnzerApiResponseErrorTransfer $unzerApiResponseErrorTransfer
+     *
+     * @return \Generated\Shared\Transfer\UnzerPaymentErrorTransfer
+     */
+    protected function createUnzerPaymentErrorTransfer(UnzerApiResponseErrorTransfer $unzerApiResponseErrorTransfer): UnzerPaymentErrorTransfer
```

Comment:
please rename method to `mapATransferToBTransfer` and add 2nd argument
then pass empty transfer when needed

Reply:
Fixed.